docs/meshstack.openstack.index.md

```diff
@@ -48,3 +48,9 @@ Project Users get the following OpenStack roles:
 The actual access rights associated with these roles are managed by the OpenStack instance and are not part of meshStack.
 
 More details about the User Federation with OpenStack can be found [here](meshstack.openstack.index.md).
+
+## Federated Idp
+
+meshIdB is not required for the integration anymore. You can directly integrate your company-wide IdP to OpenStack. To do this its necessary to enable the federatedIdp within the OpenStack plaform configuration in meshfed. Besides that the Idp needs to be fully integrated within OpenStack. This means users should have access to OpenStack independent of the panel login. During the automated replication meshStack will make sure that users have access to the OpenStack projects they are assigned to in meshStack. Every new created User in OpenStack will be assigned to the MeshUsers domain to prevent the creation of one user multiple times. If the user still exists in OpenStack then the automatisation will pick this user to assign him to the specific groups. We create for each project an own group.
```

Comment:
d: But how does meshStack have to be configured? If meshDelivery looks into the docs they will not know and ask us. Therefore it is important to actually show what needs to be configured. So we need an according dhall snippet in here that shows that you have to set this flag to activate it and perhaps more if there is more to configure. 

And on OpenStack side it would be helpful to have a snippet of the mapping that must be done there. You can request that from @henryde.

And in general explain that this mode is an alterantive to the previously described meshIDB integration. And it will be the superior integration in future. The old one will be removed in future. Therefore below ## Authorization there should be two sections. meshIdP and Enterprise IdP or similar. Because these are the two options that can be used for AuthZ

Reply:
I don't agree that we need now a separate section. I think its ok to mention that this is an alternative against the old one. 
I've append the dhall configuration.

Reply:
I think it still makes sense to have a "Authentication & Authorization" section with 2 sub-sections "meshIdP" and "Enterprise IdP" or similar.

Reply:
I've added this.